plugin/storage/grpc/shared/grpc_handler.go

```diff
@@ -52,7 +52,7 @@ func NewGRPCHandler(impl *GRPCHandlerStorageImpl) *GRPCHandler {
 // NewGRPCHandler creates a handler given implementations grouped by plugin services.
 func NewGRPCHandlerWithPlugins(
```

Comment:
do we need this? I don't see anything using this, I was expecting the remote-storage to utilize this framework, but if it doesn't then we can delete it.

Reply:
(maybe in a prequel PR)

Reply:
https://github.com/jaegertracing/jaeger/pull/6580